src/main/webapp/src/app/app.component.html

```diff
@@ -1,6 +1,6 @@
 <!-- Template page: these components will appear on
 every page unless they are suppressed. -->
-<mat-toolbar id="growpod-top-bar" color="primary">
+<mat-toolbar>
```

Comment:
If the id `growpod-top-bar` is not used anymore I'd recommend cleaning up references to it elsewhere if they exist, i.e. in css or js.